bundles/org.openhab.core.config.discovery/src/main/java/org/openhab/core/config/discovery/internal/DiscoveryServiceRegistryImpl.java

```diff
@@ -189,27 +190,27 @@ public void addDiscoveryListener(DiscoveryListener listener) throws IllegalState
     }
 
     @Override
-    public boolean startScan(ThingTypeUID thingTypeUID, @Nullable ScanListener listener) throws IllegalStateException {
+    public boolean startScan(ThingTypeUID thingTypeUID, @Nullable String input, @Nullable ScanListener listener) {
         Set<DiscoveryService> discoveryServicesForThingType = getDiscoveryServices(thingTypeUID);
 
         if (discoveryServicesForThingType.isEmpty()) {
             logger.warn("No discovery service for thing type '{}' found!", thingTypeUID);
             return false;
         }
 
-        return startScans(discoveryServicesForThingType, listener);
+        return startScans(discoveryServicesForThingType, input, listener);
     }
 
     @Override
-    public boolean startScan(String bindingId, final @Nullable ScanListener listener) throws IllegalStateException {
+    public boolean startScan(String bindingId, @Nullable String input, @Nullable ScanListener listener) {
```

Comment:
could you pleas explain why you removed the throws declaration? ```getDiscoveryServices``` is still used and declares this exception....

Reply:
Fixed